### PR TITLE
DOCS-10756 Accessibility

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
DOCS-10756 Accessibility

### CODE REVIEW

**Review by ferferrarazzo — DISMISSED** [review]

File: modules/ROOT/pages/accessibility.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+= Accessibility
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+All Anypoint API Community Manager components, including both components provided by MuleSoft and components in Salesforce Experience Cloud, comply with the Web Content Accessibility Guidelines (WCAG) 2.1 AA level accessibility standard, and enable the creation of sites compliant with this standard and level.
```
Comment (kbostrom) [inline modules/ROOT/pages/accessibility.adoc]:
```suggestion
All Anypoint API Community Manager components, including both components provided by MuleSoft and components in Salesforce Experience Cloud, comply with the Web Content Accessibility Guidelines (WCAG) 2.1 AA-level accessibility standard and enable the creation of sites compliant with this standard and level.
```

File: modules/ROOT/pages/accessibility.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Because MuleSoft customers design and brand their API communities, accessibility assessment for sites built with API Community Manager depends on the customer's choice of design, structure, look and feel, visual, user interface, and behavior elements. Customers are responsible for accessibility assessment of these aspects on their specific community.
```
Comment (kbostrom) [inline modules/ROOT/pages/accessibility.adoc]: "in" a community rather than "on"

Comment (kbostrom) [reply modules/ROOT/pages/accessibility.adoc]: Also, take a look at the list of dependencies: for example, is it redundant to mention "look and feel" and then also "visual" and "user interface"?

**Review by kbostrom — DISMISSED** [review]
CE

File: modules/ROOT/pages/accessibility.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/accessibility.adoc]: Fixed.

File: modules/ROOT/pages/accessibility.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/accessibility.adoc]: Agreed, better to trim that list. Done. Also changed "on" to "in."

**Review by kbostrom — APPROVED** [review]
RA

**Pull request merged**